os/mm/kmm_heap/kmm_calloc.c

```diff
@@ -114,18 +114,22 @@ void *kmm_calloc_at(int heap_index, size_t n, size_t elem_size)
 		return NULL;
 	}
 
+	if (n * elem_size == 0) {
```

Comment:
instead of multiply, I think below is better.
```
if (n == 0 || elem_size == 0) {
     mdbg("wrong arg, n : %d elem_size : %d\n");
     return NULL;
}
```
BTW, in this situation you don't have to set errno as -EINVAL?

Reply:
It is better to set the errno, but there is no errno setting in all mm.
I will consider it later.
Thanks.